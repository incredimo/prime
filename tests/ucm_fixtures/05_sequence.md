```ucm get#g1 path="./in.txt"```
```ucm run#r1 lang="python"```