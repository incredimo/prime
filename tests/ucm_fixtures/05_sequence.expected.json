{
  "sequence": [
    "g1",
    "r1"
  ]
}